Library/Formula/macvim.rb

```diff
@@ -67,6 +67,15 @@ def install
     elsif build.with? "python"
       ENV.prepend "LDFLAGS", `python-config --ldflags`.chomp
       ENV.prepend "CFLAGS", `python-config --cflags`.chomp
+      framework_script = <<-EOS.undent
+        import distutils.sysconfig
+        print distutils.sysconfig.get_config_var("PYTHONFRAMEWORKPREFIX")
```

Comment:
Any distribution of Python 2.7. We should have 2.7 because of the `:python` dependency. This var is useful for framework builds; I tested that a non-framework build installed by pyenv returns the empty string (not None).